File: class8/ex33_totalSalaries.c
/*Suponha uma empresa com 5 funcionários onde cada um tem um
salário de 100, 50, 50, 120 e 30 reais respectivamente. Quanto a
empresa gasta em salários no total?*/

#include <stdio.h>

int main() {

    int salaries[5] = {100, 50, 50, 120, 30};
    int totalSalaries = 0; 

    for (int i = 0; i < 5; i++) {
        
        totalSalaries += salaries[i];
    }

    printf("Salarios ao total: %d\n", totalSalaries);

    return 0;
}

File: GO/class8/ex33_totalSalaries.go
/*Suponha uma empresa com 5 funcionários onde cada um tem um
salário de 100, 50, 50, 120 e 30 reais respectivamente. Quanto a
empresa gasta em salários no total?*/

package main

import "fmt"

func main() {
	salaries := [5]int{100, 50, 50, 120, 30}
	totalSalaries := 0

	for i := 0; i < 5; i++ {

		totalSalaries += salaries[i]
	}

	fmt.Printf("Salarios ao total: %d\n", totalSalaries)
}
